README.md

```diff
@@ -188,6 +188,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [x] [bibtex](https://github.com/latex-lsp/tree-sitter-bibtex) (maintained by @theHamsta, @clason)
 - [x] [bicep](https://github.com/amaanq/tree-sitter-bicep) (maintained by @amaanq)
 - [x] [bitbake](https://github.com/amaanq/tree-sitter-bitbake) (maintained by @amaanq)
+- [x] [blade](https://github.com/EmranMR/tree-sitter-blade) (experimental, maintained by @calebdw)
```

Comment:
Sorry, but we no longer accept experimental grammars. Either it is ready for production use, or it needs to cook outside of nvim-treesitter. (We have too many languages now that frequent breaking changes are too much to handle for me.)

Reply:
I understand, I've held off on adding this grammar for that reason---but this has been around since 2023 and has been pretty stable so figured it's time to add it.

Wasn't sure what the experimental flag was for so just added it to be safe but I'll remove